src/syft/ast/klass.py

```diff
@@ -91,8 +98,23 @@ def run_class_method(
             if attr_path_and_name is not None:
```

Comment:
NITPICK: If attr_path_and name (without is not None) (I know this PR has nothing to do with this :D)

Reply:
I have previously commented this code with:
# QUESTION: Could path_and_name be None?

But i'm not sure if thats what you are referring to?

Reply:
Ah...I am referring to:
if attr_path_and_name <-- returns false if attr_path_and_name is None, "" (empty string), [] (empty list) and 0.
I am ok to leave it like it is now, I was just asking if it would make sense to make it shorter.

Reply:
**LE**: Thinking more into this. I think we should be precise on what we check against -- we need to *clearly* check it against None

Reply:
This was a good catch since it prompted me to look at the code closer and remove a lot of unnecessary branching and checks since args will always be a list / tuple even if its just an empty one.